input/pagecontent/reference.md

```diff
@@ -1653,6 +1650,70 @@ A filter is a logical statement in the form `{property} {operator} {value}`, whe
 
 > **Note:** Intensional and extensional forms can be used together in a single value set definition.
 
+{%include tu-div.html%}
+
+##### Concept Metadata
+
+Within a ValueSet definition, the caret syntax MAY be used to set metadata attributes for individual concepts (e.g., elements of ValueSet.compose.include.concept.designation).
+
+To assign metadata values for concepts that are included in the value set, authors SHOULD specify one or more indented caret path assignment rules below the rule that includes the concept:
+<pre><code>* <span class="optional">include</span> {Coding}
+  * ^&lt;element1 of corresponding concept&gt; = {value1}
+  <span class="optional">* ^&lt;element2 of corresponding concept&gt; = {value2}</span>
+</code></pre>
+
+To assign metadata values for concepts that are excluded in the value set, authors SHOULD specify one or more indented caret path assignment rules below the rule that excludes the concept:
+<pre><code>* exclude {Coding}
+  * ^&lt;element1 of corresponding concept&gt; = {value1}
+  <span class="optional">* ^&lt;element2 of corresponding concept&gt; = {value2}</span>
+</code></pre>
+
+If authors do not wish to use indented caret path rules, they MAY specify the code followed by a caret path assignment:
+<pre><code>* {Coding} ^&lt;element1 of corresponding concept&gt; = {value1}
+<span class="optional">* {Coding} ^&lt;element2 of corresponding concept&gt; = {value2}</span>
+</code></pre>
+
+> **Note:** A concept must be included or excluded _before_ caret rule assignments can be used to set its metadata. When a single rule contains a concept followed by a caret path assignment, the assignment pertains to the concept wherever it is found in the composition (whether included or excluded).
+
+**Examples:**
+
+* To set the designation.use of the included concept `$SCT#971000205103`:
+
+  ```
+  * $SCT#971000205103 "Wearing street clothes with shoes"
+    * ^designation[0].use = $SCT#900000000000003001 "Fully specified name"
+  ```
+
+* To set the designation.use of the included concept `$SCT#971000205103` using the `include` keyword:
```

Comment:
Can you show a bunch of different metadata elements in these examples?

Reply:
Yeah. I was just using the same basic example as in the CodeSystem. This just means I need to go and find realistic uses of `designation` in a ValueSet -- because I don't have confidence that anything I make up will make any sense.

Reply:
Give it a try at least.